src/taskProvider.ts

```diff
@@ -319,36 +330,56 @@ export class TaskProvider implements vscode.TaskProvider {
 
   private appLoadExec(): string {
     let exec = "";
+    let keyconfig = "";
     if (platform === "linux") {
       // Linux
+      if (this.scpConfig === true) {
+        keyconfig = "-e SCP_PRIVKEY=${SCP_PRIVKEY}";
+      }
       // Executes make load in the container to load the app on a physical device.
-      exec = `docker exec -it  ${this.containerName} bash -c 'export BOLOS_SDK=$(echo ${getSelectedSDK()}) && make load'`;
+      exec = `docker exec -it ${keyconfig} ${this.containerName} bash -c 'export BOLOS_SDK=$(echo ${getSelectedSDK()}) && make load'`;
     } else if (platform === "darwin") {
       // macOS
+      if (this.scpConfig === true) {
+        keyconfig = "--rootPrivateKey ${SCP_PRIVKEY}";
```

Comment:
Did you try it ? Not sure there would be proper variable expansion :thinking:

Reply:
I don't have a Mac to test (neither Windows). I just kept the same syntax as the other variables in the exec command, like `${this.appName}`

Reply:
`${<variable>}` is for expanding program variables when using ` ` ` quotes around a string. Not for expanding host's environment variables. We should test at least on one of the other platforms (either windows or mac - on a VM if needed) before releasing. This is painful but one of the goals of this extension is to be cross platform.

Reply:
For Windows there are official virtual machine images available on Microsoft's website : https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/ (I tried the vmware image and it works Ok on linux with vmware player)

Reply:
Done. tested and it works also with Windows

Reply:
Tested also OK on Mac